ci-test-files/centos-stream9-imagestream.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   - annotations: null
     from:
       kind: DockerImage
-      name: quay.io/kubevirt/centos8-container-disk-images
+      name: quay.io/containerdisks/centos-stream
     importPolicy:
       scheduled: true
-    name: latest
+    name: 9
```

Comment:
HCO only supports `latest` because this is what CDI supports. This is not going to work without changing the imagestream related code that forces it.

Reply:
@arnongilboa ^^^

Reply:
There is a bug in this logic in HCO, so it's not blocking this imagestream. However, CDI can't handle it. From CDI log inhttps://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/kubevirt_hyperconverged-cluster-operator/1695/pull-ci-kubevirt-hyperconverged-cluster-operator-main-hco-e2e-image-index-aws/1480659447293415424/artifacts/hco-e2e-image-index-aws/gather-extra/artifacts/pods/kubevirt-hyperconverged_cdi-deployment-6bb9c747c8-mcwwg_cdi-controller.log

```
{"level":"error","ts":1641855138.6631045,"logger":"controller-runtime.manager.controller.dataimportcron-controller","msg":"Reconciler error","name":"centos-stream9-image-cron-is","namespace":"kubevirt-os-images","error":"No imagestream tag items centos-stream9","errorVerbose":"No imagestream tag items centos-stream9\nkubevirt.io/containerized-data-importer/pkg/controller.getImageStreamDigest\n\tpkg/controller/dataimportcron-controller.go:145\nkubevirt.io/containerized-data-importer/pkg/controller.(*DataImportCronReconciler).updateImageStreamDesiredDigest\n\tpkg/controller/dataimportcron-controller.go:291\nkubevirt.io/containerized-data-importer/pkg/controller.(*DataImportCronReconciler).pollImageStreamDigest\n\tpkg/controller/dataimportcron-controller.go:157\nkubevirt.io/containerized-data-importer/pkg/controller.(*DataImportCronReconciler).update\n\tpkg/controller/dataimportcron-controller.go:247\nkubevirt.io/containerized-data-importer/pkg/controller.(*DataImportCronReconciler).Reconcile\n\tpkg/controller/dataimportcron-controller.go:100\nkubevirt.io/containerized-data-importer/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\tvendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:298\nkubevirt.io/containerized-data-importer/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\tvendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:253\nkubevirt.io/containerized-data-importer/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.2\n\tvendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:216\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.BackoffUntil\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.UntilWithContext\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:99\nruntime.goexit\n\tGOROOT/src/runtime/asm_amd64.s:1371","stacktrace":"kubevirt.io/containerized-data-importer/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\tvendor/github.com/go-logr/zapr/zapr.go:132\nkubevirt.io/containerized-data-importer/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\tvendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:302\nkubevirt.io/containerized-data-importer/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\tvendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:253\nkubevirt.io/containerized-data-importer/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.2\n\tvendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:216\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.BackoffUntil\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185\nkubevirt.io/containerized-data-importer/vendor/k8s.io/apimachinery/pkg/util/wait.UntilWithContext\n\tvendor/k8s.io/apimachinery/pkg/util/wait/wait.go:99"}
```